src/components/Account-panel/Order-history/OrdersList.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  Dispatch,
+  FC,
+  Fragment,
+  SetStateAction,
+  useCallback,
+  useEffect,
+  useRef
+} from 'react'
+import { Box, Divider } from '@mui/material'
+import { format } from 'date-fns'
+import { uk } from 'date-fns/locale'
+import { Typography } from '../../UI/Typography'
+import { Status, StepCustom } from '../../Step/Step'
+import { useTheme } from '@mui/material/styles'
+
+export type OrdersType = {
+  ordered_products: any[]
```

Comment:
do not use any

Reply:
and also, maybe you already have this type from BE?

Reply:
done